python/tskit/tables.py

```diff
@@ -335,16 +336,41 @@ def __setattr__(self, name, value):
 
     def __getitem__(self, index):
         """
-        Return the specified row of this table, decoding metadata if it is present.
-        Supports negative indexing, e.g. ``table[-5]``.
+        If passed an integer, return the specified row of this table, decoding metadata
+        if it is present. Supports negative indexing, e.g. ``table[-5]``.
+        If passed a slice, iterable or array return a new table containing the specified
+        rows. Similar to numpy fancy indexing, if the array or iterables contains
+        booleans then in acts as a mask, returning those rows for which are True.
```

Comment:
s/in/it

Reply:
The sentence isn't quite grammatical, something like "returning those rows for which the mask is True."

Reply:
Fixed